code/modules/cargo/packs/security.dm

```diff
@@ -207,3 +212,19 @@
 					/obj/item/storage/box/wall_flash,
 					/obj/item/storage/box/wall_flash)
 	crate_name = "wall-mounted flash crate"
+
+/datum/supply_pack/security/hunting
```

Comment:
Quick question, is their now two locations you can buy this pack?
I thought its placed in organic still now here as well?